lib/core/test.js

```diff
@@ -86,7 +89,11 @@ Test.prototype.deployAll = function(contractsConfig, cb) {
       }
       self.web3.eth.defaultAccount = accounts[0];
       RunCode.doEval(result, self.web3); // jshint ignore:line
-      cb();
+      if (self.options.logging) {
+        cb(logger);
```

Comment:
Yeah, that would work as well.  It was just the most convenient option for me at the time, to get ahold of that logger within the test context (mostly to just `console.log` it).